package.json

```diff
@@ -22,8 +22,9 @@
     "start": "node destTransformer.js",
     "testDestinations": "npx jest --runInBand --detectOpenHandles --coverage --notify",
     "testVersionedRouter": "npx jest versionedRouter --type=all --detectOpenHandles --notify",
-    "testVersioned": "npx jest versionedRouter --destName=braze --type=batch",
-    "verify": "eslint . || exit 1; jest --runInBand --detectOpenHandles --coverage --notify || exit 1"
+    "testVersioned": "npx jest versionedRouter --destName=heap --type=all --notify",
+    "verify": "eslint . || exit 1; jest --runInBand --detectOpenHandles --coverage --notify || exit 1",
+    "coverage": "npx jest --runInBand --detectOpenHandles --coverage --notify && npx jest versionedRouter --destName=heap --type=all --detectOpenHandles  --notify"
```

Comment:
> npx jest versionedRouter --destName=heap --type=all --detectOpenHandles  --notify

Why do we need this for heap specifically ?

Reply:
We have written the versionedRouter test for heap as the default destination. This was original implementation of versionedRouter test